### PR TITLE
Add pressure convergence plot for Jeffery-Hamel flow.

### PR DESCRIPTION
**Pull request by jwpeterson**
Previously we demonstrated 3rd-order (optimal) convergence rates for
the velocity components, this PR shows that we also get 2nd-order
(optimal) convergence rates for pressure.

Refs #3036, #7904.

Updated convergence plot:
[jeffery_hamel.pdf](https://github.com/idaholab/moose/files/557127/jeffery_hamel.pdf)

### CODE REVIEW

File: modules/navier_stokes/src/tests/WedgeFunction.C [hunk 1]
```diff
@@ -13,11 +13,12 @@ InputParameters validParams<WedgeFunction>()
   InputParameters params = validParams<Function>();
   params.addRequiredParam<Real>("alpha_degrees", "The wedge half-angle size (in degrees) used in computing 'f' below.");
   params.addRequiredParam<Real>("Re", "The Reynolds number used in computing 'f' below.");
-  params.addRequiredParam<unsigned int>("component", "The component (x==0, y==1) of the velocity we are computing.");
+  params.addRequiredRangeCheckedParam<unsigned int>("var_num", "var_num<3", "The variable (0=vel_x, 1==vel_y, 2==p) we are computing the exact solution for.");
```
Comment (andrsd) [inline modules/navier_stokes/src/tests/WedgeFunction.C]: A mix of `=` and `==` in the doco string

File: modules/navier_stokes/tests/ins/jeffery_hamel/jeffery_hamel.py [hunk 2]
```diff
@@ -185,11 +192,42 @@ def solve_ode(guess, dt, alpha, Re):
 if (current_iterate >= max_its):
   print('\nWarning, max iterates reached before reaching tolerance!\n')
 
+# Compute the "constant" K for each value of eta.  It turns out it is
+# not actually constant.. but it is fairly close.  We can compute a
+# mean and standard deviation to try and pick the best single value...
+K = []
+for i in xrange(len(timesteps)):
+  K.append(-1./(4. * alpha * alpha) * (alpha * Re * solution[i]**2 + second_deriv[i]) - solution[i])
```
Comment (andrsd) [inline modules/navier_stokes/tests/ins/jeffery_hamel/jeffery_hamel.py]: spaces around `/`

File: modules/navier_stokes/src/tests/WedgeFunction.C (hunk 1 above)
Comment (andrsd) [inline modules/navier_stokes/src/tests/WedgeFunction.C]: Maybe even consider using `MooseEnum` of it makes more sense...

**Review by andrsd — CHANGES_REQUESTED** [review]

File: modules/navier_stokes/src/tests/WedgeFunction.C (hunk 1 above)
Comment (jwpeterson) [inline modules/navier_stokes/src/tests/WedgeFunction.C]: I thought about that, but this is only for a test, and I also use it to index into a vector, so I think it makes more sense as an int.

**Review by andrsd — APPROVED** [review]

**Pull request merged**